platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/components/LinkedEventIntake.java

```diff
@@ -85,46 +74,28 @@ public class LinkedEventIntake {
      * Constructor
      *
      * @param platformContext    the platform context
-     * @param threadManager      creates new threading resources
      * @param time               provides the wall clock time
      * @param consensusSupplier  provides the current consensus instance
      * @param dispatcher         invokes event related callbacks
      * @param shadowGraph        tracks events in the hashgraph
-     * @param prehandleEvent     prehandles transactions in an event
      * @param intakeEventCounter tracks the number of events from each peer that are currently in the intake pipeline
      */
     public LinkedEventIntake(
             @NonNull final PlatformContext platformContext,
-            @NonNull final ThreadManager threadManager,
             @NonNull final Time time,
             @NonNull final Supplier<Consensus> consensusSupplier,
             @NonNull final EventObserverDispatcher dispatcher,
             @NonNull final ShadowGraph shadowGraph,
-            @NonNull final Consumer<EventImpl> prehandleEvent,
             @NonNull final IntakeEventCounter intakeEventCounter) {
 
         this.time = Objects.requireNonNull(time);
         this.consensusSupplier = Objects.requireNonNull(consensusSupplier);
         this.dispatcher = Objects.requireNonNull(dispatcher);
         this.shadowGraph = Objects.requireNonNull(shadowGraph);
-        this.prehandleEvent = Objects.requireNonNull(prehandleEvent);
         this.intakeEventCounter = Objects.requireNonNull(intakeEventCounter);
 
         this.paused = false;
-
-        final EventConfig eventConfig = platformContext.getConfiguration().getConfigData(EventConfig.class);
-
-        final BlockingQueue<Runnable> prehandlePoolQueue = new LinkedBlockingQueue<>();
-
-        prehandlePool = new ThreadPoolExecutor(
-                eventConfig.prehandlePoolSize(),
-                eventConfig.prehandlePoolSize(),
-                0L,
-                TimeUnit.MILLISECONDS,
-                prehandlePoolQueue,
-                threadManager.createThreadFactory("platform", "txn-prehandle"));
-
-        metrics = new EventIntakeMetrics(platformContext, prehandlePoolQueue::size);
+        metrics = new EventIntakeMetrics(platformContext, () -> 0 /* TODO */);
```

Comment:
TODO

Reply:
TODO removed. This metric will simply not report a real value with the new intake (automatic metric will capture the queue size).